### DISCUSSION TITLE
Dani Subject @ IBM Canada

### DISCUSSION THREAD

**Discussion by dsubject** [Green-Software-Foundation/discussions#174] [Introductions]
Hi everyone, my name's Danielle but I go by Dani (she/her). I'm a Senior Developer at IBM, over 6 years of experience in software development, mainly frontend (Angular/React/Native Android). 

I'm here because I've always been an environmentalist and when I discovered the intersection of climate awareness and software I couldn't not become involved. I often feel like I'm yelling into the ether with nobody listening when it comes to green software. I'm desperate to meet other like-minded folks who I can connect with on this and hopefully become involved in a project of some sort. If anyone is looking for a developer to help with their green software project please ping me. 

I am active on Github, but you can also find me on [LinkedIn](https://www.linkedin.com/in/danielle-subject/) or send me an email daniellesubject@gmail.com

**Comment by NAMRATA-WOKE** [comment]
Hi @dsubject! Welcome. Did you know IBM is a member of the GSF? That gives you the opportunity to collaborate with over 700 practitioners within our ecosystem working on building new standards and tooling to advance green software. If you're interested in participating in our working groups, **[register and subscribe](https://greensoftware.foundation/onboarding/)** to the working groups that are of interest to you. 

You can also learn more about our projects [here](https://drive.google.com/file/d/1UQpO8I_v-M32y-0Yj3BduLceCpuxqcHB/view?usp=drive_link).
